### PR TITLE
Initial work on pgplot recipe.

### PR DESCRIPTION
**Pull request by pkgw**

### CODE REVIEW

**Comment by pkgw** [comment]
This is blocked on real X11 for OSX.

**Comment by pkgw** [comment]
Kicking CI to get Travis to see the freshest commit (I force-pushed).

**Comment by pkgw** [comment]
Woohoo, first example of an X11-based package building on OSX with the X.org libraries! This should be mergeable now. (At the time of writing, the AppVeyor build hasn't run yet but we don't attempt to build on Windows and the "build" succeeded last time, so ...)

File: recipes/pgplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - test -x $PREFIX/bin/pgxwin_server
+    - test -f $PREFIX/lib/libpgplot.a  # [unix]
+    - test -f $PREFIX/lib/libpgplot.so  # [linux]
+    - test -f $PREFIX/lib/libpgplot.dylib  # [osx]
+    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
+    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+
+about:
+  home: http://www.astro.caltech.edu/~tjp/pgplot/
+  license: non-commercial use only
```
Comment (ocefpaf) [inline recipes/pgplot/meta.yaml]: Do we have a license file to clarify that? (Can be done in the feedstock.)

**Pull request merged**